cluster/search/load_data.py

```diff
@@ -35,21 +38,27 @@
 # Indico
 indicoio.config.cloud = 'themeextraction'
 
-with open('cluster/search/sp500.txt') as f:
-    SP_TICKERS = f.read().splitlines()
+with open(os.path.join(os.path.dirname(__file__), "data", "sp500.txt"), 'rb') as f:
```

Comment:
Totally nitpicking but it's kind of weird that we use a context manager for one of these and a regular read for the other, haha.